problems/[230717] 739. Daily Temperatures.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/daily-temperatures
+ */
+
+/**
+ * @param {number[]} temperatures
+ * @return {number[]}
+ */
+const dailyTemperatures = function (temperatures) {
+  const result = [];
```

Comment:
```suggestion
  const result = new Array(temperatures.length).fill(0);
```
이렇게 하면 메모리를 미리 할당해 요소가 추가될 때마다 발생하는 오버헤드를 막을 수 있을것 같습니다.